src/System.Net.Primitives/pkg/System.Net.Primitives.pkgproj

```diff
@@ -3,7 +3,11 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.Net.Primitives.csproj">
-      <SupportedFramework>net46;netcore50;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
+      <SupportedFramework>net463;netcoreapp1.1;$(AllXamarinFrameworks)</SupportedFramework>
+    </ProjectReference>
+    <ProjectReference Include="..\src\System.Net.Primitives.csproj">
+      <OSGroup>Windows_NT</OSGroup>
+      <TargetGroup>net463</TargetGroup>
```

Comment:
Small NIT: Can you put TargetGroup before OSGroup? we are trying to get all of our builds files consistent, and that's the order we have across the repo.